content/en/account_management/guide/relevant-usage-migration.md

```diff
@@ -0,0 +1,234 @@
+---
+title: Relevant Usage Migration Guide
+---
+
+## Overview
+
+On October 1, 2024, two API endpoints will change:
+- [Get hourly usage by product family][1]
+- [Get usage across your account][2]
+
+The RUM and Indexed Logs products are affected. See the following section for each API endpoint you use, and review the upcoming updates to determine what changes to make to your automation. 
+
+## Get hourly usage by product family
+
+The October 1, 2024 change to the [Get hourly usage by product family][1] endpoint deprecates legacy keys and provides more granular information about your RUM usage.
+
+The following new keys describe RUM usage:
+- `rum_replay_session_count`
+- `rum_lite_session_count`
+- `rum_browser_legacy_session_count`
+- `rum_browser_replay_session_count`
+- `rum_browser_lite_session_count`
+- `rum_mobile_legacy_session_count_android`
+- `rum_mobile_legacy_session_count_flutter`
+- `rum_mobile_legacy_session_count_ios`
+- `rum_mobile_legacy_session_count_reactnative`
+- `rum_mobile_legacy_session_count_roku`
+- `rum_mobile_lite_session_count_android`
+- `rum_mobile_lite_session_count_flutter`
+- `rum_mobile_lite_session_count_ios`
+- `rum_mobile_lite_session_count_reactnative`
+- `rum_mobile_lite_session_count_roku`
+
+### RUM
+
+RUM usage in the [Get hourly usage by product family][1] v2 endpoint will be shown under one RUM product family with three keys representing SKUs that your RUM usage may be billed on:
+- `rum`
+- `rum_replay`
+- `rum_lite`
+
+ Legacy product families and usage types will be deprecated and displayed as null in the Hourly Usage by Product Family v2 endpoint.
+
+ #### Current response structure
+
+ A current example of a response for the [Get hourly usage by product family][1] v2 endpoint follows:
```

Comment:
Can you add examples of the upcoming response structures in all places as well so we can show the delta?